2022-04-05 14:43:52,054 INFO    Thread-30 :6228 [wandb_setup.py:_flush():75] Loading settings from C:\Users\adria\.config\wandb\settings
2022-04-05 14:43:52,055 INFO    Thread-30 :6228 [wandb_setup.py:_flush():75] Loading settings from wandb\settings
2022-04-05 14:43:52,055 INFO    Thread-30 :6228 [wandb_setup.py:_flush():75] Loading settings from environment variables: {'project': 'pw05', 'entity': 'awilli', 'root_dir': 'C:\\Users\\adria\\OneDrive\\Dokumente\\Studium - Hslu\\Master\\TSM_DeLearn\\Practical Work 05', 'run_id': 'qa1v56ar', 'sweep_param_path': 'C:\\Users\\adria\\OneDrive\\Dokumente\\Studium - Hslu\\Master\\TSM_DeLearn\\Practical Work 05\\wandb\\sweep-bg0g4ucn\\config-qa1v56ar.yaml', 'sweep_id': 'bg0g4ucn'}
2022-04-05 14:43:52,055 INFO    Thread-30 :6228 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-04-05 14:43:52,056 INFO    Thread-30 :6228 [wandb_init.py:_log_setup():405] Logging user logs to C:\Users\adria\OneDrive\Dokumente\Studium - Hslu\Master\TSM_DeLearn\Practical Work 05\wandb\run-20220405_144352-qa1v56ar\logs\debug.log
2022-04-05 14:43:52,056 INFO    Thread-30 :6228 [wandb_init.py:_log_setup():406] Logging internal logs to C:\Users\adria\OneDrive\Dokumente\Studium - Hslu\Master\TSM_DeLearn\Practical Work 05\wandb\run-20220405_144352-qa1v56ar\logs\debug-internal.log
2022-04-05 14:43:52,056 INFO    Thread-30 :6228 [wandb_init.py:_jupyter_setup():355] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000002365C0D48B0>
2022-04-05 14:43:52,057 INFO    Thread-30 :6228 [wandb_init.py:init():439] calling init triggers
2022-04-05 14:43:52,057 INFO    Thread-30 :6228 [wandb_init.py:init():442] wandb.init called with sweep_config: {'batch_size': 64, 'epochs': 10, 'learning_rate': 0.001}
config: {}
2022-04-05 14:43:52,057 INFO    Thread-30 :6228 [wandb_init.py:init():492] starting backend
2022-04-05 14:43:52,057 INFO    Thread-30 :6228 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=spawn, using: spawn
2022-04-05 14:43:52,077 INFO    Thread-30 :6228 [backend.py:ensure_launched():219] starting backend process...
2022-04-05 14:43:52,245 INFO    Thread-30 :6228 [backend.py:ensure_launched():224] started backend process with pid: 17616
2022-04-05 14:43:52,246 INFO    Thread-30 :6228 [wandb_init.py:init():501] backend started and connected
2022-04-05 14:43:52,277 INFO    Thread-30 :6228 [wandb_run.py:_config_callback():992] config_cb None None {'batch_size': 64, 'epochs': 10, 'learning_rate': 0.001}
2022-04-05 14:43:52,278 INFO    Thread-30 :6228 [wandb_run.py:_label_probe_notebook():947] probe notebook
2022-04-05 14:43:52,278 INFO    Thread-30 :6228 [wandb_run.py:_label_probe_notebook():957] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-04-05 14:43:52,278 INFO    Thread-30 :6228 [wandb_init.py:init():565] updated telemetry
2022-04-05 14:43:52,355 INFO    Thread-30 :6228 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-04-05 14:43:55,253 INFO    Thread-30 :6228 [wandb_run.py:_on_init():1759] communicating current version
2022-04-05 14:43:55,993 INFO    Thread-30 :6228 [wandb_run.py:_on_init():1763] got version response 
2022-04-05 14:43:55,993 INFO    Thread-30 :6228 [wandb_init.py:init():625] starting run threads in backend
2022-04-05 14:44:01,028 INFO    Thread-30 :6228 [wandb_run.py:_console_start():1733] atexit reg
2022-04-05 14:44:01,031 INFO    Thread-30 :6228 [wandb_run.py:_redirect():1606] redirect: SettingsConsole.WRAP
2022-04-05 14:44:01,031 INFO    Thread-30 :6228 [wandb_run.py:_redirect():1643] Wrapping output streams.
2022-04-05 14:44:01,045 INFO    Thread-30 :6228 [wandb_run.py:_redirect():1667] Redirects installed.
2022-04-05 14:44:01,046 INFO    Thread-30 :6228 [wandb_init.py:init():664] run started, returning control to user process
2022-04-05 14:44:01,047 INFO    Thread-30 :6228 [wandb_run.py:finish():1539] finishing run awilli/pw05/qa1v56ar
2022-04-05 14:44:01,047 INFO    Thread-30 :6228 [jupyter.py:save_history():429] not saving jupyter history
2022-04-05 14:44:01,048 INFO    Thread-30 :6228 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-04-05 14:44:01,048 INFO    Thread-30 :6228 [wandb_init.py:_jupyter_teardown():337] cleaning up jupyter logic
2022-04-05 14:44:01,048 INFO    Thread-30 :6228 [wandb_run.py:_atexit_cleanup():1702] got exitcode: 1
2022-04-05 14:44:01,050 INFO    Thread-30 :6228 [wandb_run.py:_restore():1674] restore
2022-04-05 14:44:03,551 INFO    Thread-30 :6228 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
}
pusher_stats {
}

2022-04-05 14:44:03,787 INFO    Thread-30 :6228 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  total_bytes: 1003
}

2022-04-05 14:44:04,255 INFO    Thread-30 :6228 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1003
  total_bytes: 1003
}

2022-04-05 14:44:04,376 INFO    Thread-30 :6228 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 1003
  total_bytes: 4117
}

2022-04-05 14:44:04,493 INFO    Thread-30 :6228 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 1003
  total_bytes: 4117
}

2022-04-05 14:44:04,604 INFO    Thread-30 :6228 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 1003
  total_bytes: 4117
}

2022-04-05 14:44:04,713 INFO    Thread-30 :6228 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 1003
  total_bytes: 4117
}

2022-04-05 14:44:04,821 INFO    Thread-30 :6228 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 1003
  total_bytes: 4117
}

2022-04-05 14:44:04,931 INFO    Thread-30 :6228 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4091
  total_bytes: 4117
}

2022-04-05 14:44:05,040 INFO    Thread-30 :6228 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4117
  total_bytes: 4117
}

2022-04-05 14:44:05,150 INFO    Thread-30 :6228 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4117
  total_bytes: 4117
}

2022-04-05 14:44:05,268 INFO    Thread-30 :6228 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4117
  total_bytes: 4117
}

2022-04-05 14:44:05,385 INFO    Thread-30 :6228 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4117
  total_bytes: 4117
}

2022-04-05 14:44:06,003 INFO    Thread-30 :6228 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4117
  total_bytes: 4117
}

2022-04-05 14:44:06,746 INFO    Thread-30 :6228 [wandb_run.py:_on_finish():1831] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4117
  total_bytes: 4117
}
local_info {
}

2022-04-05 14:44:08,950 INFO    Thread-30 :6228 [wandb_run.py:_footer_history_summary_info():2865] rendering history
2022-04-05 14:44:08,951 INFO    Thread-30 :6228 [wandb_run.py:_footer_history_summary_info():2894] rendering summary
2022-04-05 14:44:08,958 INFO    Thread-30 :6228 [wandb_run.py:_footer_sync_info():2822] logging synced files
